roles/kuryr_master/templates/cni-daemonset.yaml.j2

```diff
@@ -0,0 +1,51 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kuryr-cni-ds
+  namespace: kube-system
```

Comment:
should be "{{ kuryr_namespace }}" here?  And elsewhere in this template.

Overall, this template is not very configurable.  Is it possible to link via a comment in the top of each template where the source of this template is documented?